src/store/model/amazon.ts

```diff
@@ -51,6 +51,13 @@ export const Amazon: Store = {
 			oosLabels: ['currently unavailable'],
 			captchaLabels: ['enter the characters you see below']
 		},
+		{
+			brand: 'asus',
+			model: 'tuf non-oc',
```

Comment:
Looks like this is an OC card.

Reply:
I copied the same link twice because I'm really skilled haha.  Fixed link for non-oc.